e2e/pages/enterChildren/enterChildren.js

```diff
@@ -48,7 +48,6 @@ module.exports = {
   enterChildDetails(name, day, month, year, gender = 'Boy') {
     I.fillField(this.fields(activeChild).fullName, name);
     I.fillField(this.fields(activeChild).DOB.day, day);
-    I.fillField(this.fields(activeChild).DOB.day, day);
```

Comment:
Oh, so this is what made out test run son long :)

Reply:
that added a huge .5 seconds to the run time